kingpin/actors/rightscale/server_array.py

```diff
@@ -986,6 +990,8 @@ def _wait_until_healthy(self, array, sleep=60):
             min_count = int(array.soul['elasticity_params']
                             ['bounds']['min_count'])
 
+        replace_stranded_max = int(self._options.get('replace_stranded_max', 0))
```

Comment:
This syntax for options is deprecated. Could you switch to `self.option('<var>')` here as well as `min_count` above?